packages/nimbus/src/components/card/card.mdx

```diff
@@ -10,201 +10,394 @@ menu:
+                </Box>
+              </Flex>
+              <Text color="neutral.11">
+                You have modified products that are waiting to be published
+              </Text>
+              {/* Buttons */}
+              <Stack gap="200">
+                <Button
+                  tone="primary"
+                  variant="solid"
+                  alignSelf="start"
+                >
+                  Publish Products
+                </Button>
+                <Stack direction="horizontal" gap="300">
+                  <Button tone="primary" variant="link" size="sm">
+                    Review
+                  </Button>
+                  <Button tone="primary" variant="link" size="sm">
+                    Reject
+                  </Button>
+                  <Button tone="primary" variant="link" size="sm">
+                    See drafts
+                  </Button>
+                </Stack>
+              </Stack>
+            </Stack>
+          </Stack>
+        </Stack>
       </Card.Content>
     </Card.Root>
+
+    {/* Bottom Row Cards */}
+    <Stack direction="horizontal" gap="400">
+      {/* Products Card */}
+      <Card.Root
+        cardPadding="md"
+        borderStyle="outlined"
+        width="201.5px"
+      > {/* Half width approx */}
+        <Card.Content>
+          <Stack direction="horizontal" gap="300" alignItems="center">
+            <Box
+              alignSelf="flex-start"
+              color="primary.11"
+              boxSize="1000"
+              borderRadius="50%"
+              bg="primary.3"
+              display="flex"
+              alignItems="center"
+              justifyContent="center"
+              flexShrink="0"
+            >
+              <Icons.ViewInAr width="24px" height="24px" />
+            </Box>
+            <Text textStyle="bodyStrong" fontWeight="bold" color="primary.11">
+              Products
+            </Text>
+          </Stack>
+        </Card.Content>
+      </Card.Root>
+
+      {/* Discounts Card */}
+      <Card.Root
+        cardPadding="md"
+        borderStyle="outlined"
+        width="201.5px"
+      > {/* Half width approx */}
+        <Card.Content>
+          <Stack direction="horizontal" gap="300" alignItems="center">
+            <Box
+              alignSelf="flex-start"
+              color="primary.11"
+              boxSize="1000"
+              borderRadius="50%"
+              bg="primary.3"
+              display="flex"
+              alignItems="center"
+              justifyContent="center"
+              flexShrink="0"
+            >
+              <Icons.Discount width="24px" height="24px" />
+            </Box>
+            <Text textStyle="bodyStrong" fontWeight="bold" color="primary.11">
+              Discounts
+            </Text>
+          </Stack>
+        </Card.Content>
+      </Card.Root>
+    </Stack>
   </Stack>
-)
+);
 ```
 
-### Flexible Content
+## Properties
+
+<PropTable id="Card" />
```

Comment:
@misama-ct This isn't pulling in the props. Related to the current work you'd mentioned on Friday?